src/plugins/auto/auto_executable_network.cpp

```diff
@@ -177,11 +171,17 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
             }
         }
         return decltype(ov::available_devices)::value_type {exeDevices};
+    } else if (name == ov::model_name) {
+        return _autoSchedule->_loadContext[ACTUALDEVICE].executableNetwork->GetMetric(name);
+    } else if (name == METRIC_KEY(SUPPORTED_METRICS)) {
+        IE_SET_METRIC_RETURN(SUPPORTED_METRICS,
+                             {METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS),
+                              METRIC_KEY(SUPPORTED_METRICS),
+                              METRIC_KEY(NETWORK_NAME),
+                              METRIC_KEY(SUPPORTED_CONFIG_KEYS)});
```

Comment:
what is the value of supported_config_keys?

Reply:
"SUPPORTED_CONFIG_KEYS"
Refer to this line
https://github.com/yangwang201911/openvino/blob/8223969ed00f6df6977b24965a84be8bfaf1cd87/src/inference/include/ie/ie_plugin_config.hpp#L82